.gitignore

```diff
@@ -29,3 +29,4 @@ build/
 
 ### VS Code ###
 .vscode/
+src/main/resources/application-local.properties
```

Comment:
why is it ignored?

Reply:
is ignored because my apyKey is saved there